src/plugins/vuetify.js

```diff
@@ -11,6 +11,9 @@ const cuttleTheme = {
   colors: {
     primary: '#6202EE',
     secondary: '#FD6222',
+    newPrimary: '#E1306C',
+    newSecondary: '#5865F2',
+    disabled: '#ACACAC',
```

Comment:
Do we need a disabled color? Often just applying the `disabled` attribute will do

Reply:
So on the figma file when the button is disabled it's a grey color, but when vuetify automatically disables it, it just heavily desaturates the regular color, in this case it's a really dull pink instead of the bright pink.

Reply:
![image](https://github.com/cuttle-cards/cuttle/assets/108551827/3a73a0eb-a0da-4bca-ad4a-06eb5f3c7554)
![image](https://github.com/cuttle-cards/cuttle/assets/108551827/a4fb50f4-3a2a-4a9f-9c37-98dbe18e0d78)